todo-with-kuzzle-and-vueJS/README.fr.md

```diff
@@ -0,0 +1,601 @@
+
+Vous pouvez donc commencer par ajouter les lignes suivantes:
+```js
+import KuzzleConnect from './views/KuzzleConnect.vue';
+import store from './store';
+```
+
+Puis créer la fonction de vérification:
+
+```js
+const checkConnected = async (to, from, next) => {
+  // Récupération de la variable dans le store
+  const connection = store.state.connectedToKuzzle;
+  // Si l'on est pas connecté, on redirige vers notre page de connection qui correspond à la racine de notre site
+  if (!connection || connection === false) {
+    next('/');
+    return false;
+  }
+  next();
+  return true;
+};
+```
+
+Nous pouvons maintenant modifier la section `routes` de notre `Router` pour rediriger automatiquement vers notre page de connexion et créer la route vers notre page principale:
+
+```js
+base: process.env.BASE_URL || '/',
+routes: [
+  {
+    path: '/',
+    name: 'kuzzleConnect',
+    component: KuzzleConnect
+  },
+  {
+    path: '/home',
+    beforeEnter: checkConnected,
+    name: 'home',
+    component: Home
+  }
+]
+```
+
+## Components
+Nous allons maintenant ajouter les composants qui formeront notre page principale:
+ - `Add.vue` => barre d'ajout de tache.
+ - `ManageList.vue` => barre de création/sélection de liste.
+ - `Menu.vue` => boutons de gestion multiple des taches (tout completer, tout supprimer) et de visualisation.
+ - `ModalList.vue` => modale de création de liste.
+ - `NavBar.vue` => barre de navigation permettant l'activation ou non des notifications.
+ - `Task.vue` => ligne correspondant à une tache avec boutons pour compléter/supprimer.
+
+La construction de ces composants ne sera pas détaillée étant donné qu'ils n'ont aucune intéraction directe avec Kuzzle. Vous pouvez vous référer aux fichiers correspondants pour avoir un exemple de code.
+
+### Add
+Créez le composant `/src/components/Add.vue`
+Ce composant est assez simple. La partie template sera composée d'un champ de saisie de texte ainsi que d'un bouton. Lors du click, un signal `addTask` sera envoyé au parent de ce composant, accompagné de la saisie.
+
+### ModalList
+Créez le composant `/src/components/ModalList.vue`
+Ce composant doit contenir un champ de saisie de text ainsi qu'un bouton. Lors d'un click, un signal `create` sera envoyé au parent de ce composant, accompagné de la saisie.
+
+### ManageList
+Créez le composant `/src/components/ManageList.vue`
+Ce composant doit contenir un composant ModalList (crée ci-dessus) qui sera actif ou non selon l'état d'une variable appelée `modal` initialisée a `false`. Le signal `create` reçu par cette ModalList doit être bindé a une fonction `create` qui émettra également un signal `createList` à son parent.
+Il doit également proposer un select basé sur un tableau `lists` reçu en props, et appelant une fonction `changed` lorsque sa valeur change qui va émettre un signal `setCurrentList`. Pour finir, il doit contenir un bouton appelant une fonction `newList` qui passera la variable `modal` a `true` afin de l'activer.
```

Comment:
I don't understand a thing. This is too detailed (and, frankly, hard to comprehend for anyone not familiar with Vue), especially considering that we claim at the beginning of that section that we won't detail these modules.

Reply:
In this how-to I need to explain how to use kuzzle with VueJs, I mustn't explain how VueJs works so the reader need to know it, but I think I have to explain the  behavior of the components to help the reader to create his app !